plugins/woocommerce/src/Internal/Admin/ProductForm/Tab.php

```diff
@@ -0,0 +1,11 @@
+<?php
+/**
+ * Handles product form Tab related methods.
+ */
+
+namespace Automattic\WooCommerce\Internal\Admin\ProductForm;
+
+/**
+ * Tab class.
+ */
+class Tab extends Component {} // TODO add required arguments here?
```

Comment:
Yeah, I believe we'll want to add the `name` and `title` as required here as well.